lib/metadata.ts

```diff
@@ -7,6 +7,11 @@ export interface MetadataItem {
 }
 
 export const metadata: Record<string, MetadataItem> = {
+  "carousel": {
+    title: "Carousel",
+    icon: "AutoRenew",
+    version: "0.1.0",
```

Comment:
The plan for this round of machines is to bump the version to 0.2.0 and release them in a batch.

```suggestion
    version: "0.2.0",
```